internal/command/machine/run.go

```diff
@@ -47,16 +47,31 @@ var sharedFlags = flag.Set{
 	},
 	flag.String{
 		Name:        "size",
+		Description: "Preset guest cpu and memory for a machine, defaults to shared-cpu-1x",
+		Hidden:      true,
+	},
+	flag.String{
+		Name:        "vm-size",
 		Shorthand:   "s",
 		Description: "Preset guest cpu and memory for a machine, defaults to shared-cpu-1x",
 	},
 	flag.Int{
 		Name:        "cpus",
 		Description: "Number of CPUs",
+		Hidden:      true,
+	},
+	flag.Int{
+		Name:        "vm-cpus",
+		Description: "Number of CPUs",
 	},
 	flag.Int{
 		Name:        "memory",
 		Description: "Memory (in megabytes) to attribute to the machine",
+		Hidden:      true,
+	},
+	flag.Int{
+		Name:        "vm-memory",
```

Comment:
Would be cool to have flag aliases instead of duplicating them here

Reply:
I looked into this but this PR is already *big*, and that required an entire rework of the argument system. I got part-way through and realized I wasn't actually doing anything useful

Reply:
honestly, with #2297 merged, the only value I see in this PR is the global support for `--vm-*` flags family and its aliases. I'd also remove `--vm-cpukind` as it adds more noise than clarity and `--vm-size` serves as baseline for it.

Reply:
> honestly, with #2297 merged, the only value I see in this PR is the global support for `--vm-*` flags family and its aliases.
Yeah, I definitely see where you're coming from. The only reason I didn't close it after that got merged is for the global flags, and because the median guest calculation might be useful for other things (like scale count, maybe). that said, this _does_ also handle the edge case of service changes affecting machine creation better (imo)

> I'd also remove `--vm-cpukind` as it adds more noise than clarity and `--vm-size` serves as baseline for it.
This has already been announced to people. it'd be pretty lame to announce something then remove it the _next day_.